vis_services/tests/test_internals.py

```diff
@@ -334,21 +372,21 @@ def test_views_helper_functions(self):
             err = 'success'
             try:
                 res = make_request(request.request, "PN", required_fields)
-            except QueryException, err:
-                pass
-            except Exception, err:
-                pass
+            except QueryException as e:
+                err = e
+            except Exception as err:
+                err = e
```

Comment:
We capture the exception as `err` but then we use `e` within the code block.

Reply:
yes, it should have been Exception as e.